app/_src/guides/progressively-rolling-out-unified-resource-naming.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Progressively rolling out unified resource naming
+content_type: tutorial
+---
+
+{% capture docs %}/docs/{{ page.release }}{% endcapture %}
+{% assign Kuma = site.mesh_product_name %}
+{% assign kuma = site.mesh_install_archive_name | default: "kuma" %}
+{% assign kuma-system = site.mesh_namespace | default: "kuma-system" %}
+
+By default, Envoy resources and stats in {{ Kuma }} use mixed, legacy formats. Names often do not line up with the {{ Kuma }} resources that produced them, which makes dashboards noisy and troubleshooting slower. For example, a query like:
```

Comment:
I think we should add `KRI` section in https://kuma.io/docs/2.12.x/introduction/concepts/#resource and link in this guide. I think it should contain format with definition

Reply:
will be done in a followup PR